wifiphisher/common/attacks.py

```diff
@@ -0,0 +1,322 @@
+from wifiphisher.common.constants import *
+import
+import time, threading, socket, fcntl, struct, subprocess
```

Comment:
We need to be consistent with our coding style (PEP-8). You should change everything to single line imports.